File: MultiRubroProducts/MultiRubroProducts/DTOS/Requests/Updating/UpdateProviderRequest.cs
namespace MultiRubroProducts.DTOS.Requests.Updating
{
    public class UpdateProviderRequest
    {
        public Guid ProviderId { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Address { get; set; } = string.Empty;
        public string Cellphone { get; set; } = string.Empty;
    }
}

File: MultiRubroProyectProducts/src/main/java/com/example/multirubroproyectproducts/requests/Providers/UpdateProviderRequest.java
package com.example.multirubroproyectproducts.requests.Providers;

import jakarta.validation.constraints.NotNull;
import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

import java.util.List;
import java.util.UUID;

@Setter
@Getter
@Data
@Builder
public class  UpdateProviderRequest {
    @NotNull
    UUID id;
    String name;
    String address;
    String cellphone;
    List<UUID> products;
}
